iotedgedev/envvars.py

```diff
@@ -209,6 +207,25 @@ def save_envvar(self, key, value):
             self.output.error(f("Could not update the environment variable {key} in file {dotenv_path}"))
             sys.exit(-1)
 
+    def get_registries(self):
+        registries = {}
+        self.CONTAINER_REGISTRY = {}
+        for key in os.environ:
+            if key.startswith('CONTAINER_REGISTRY_SERVER'):
+                token = key[25:]
+                if token not in registries:
+                    registries[token] = {}
+                registries[token]['server'] = os.environ[key]
+            elif key.startswith(('CONTAINER_REGISTRY_USERNAME', 'CONTAINER_REGISTRY_PASSWORD')):
+                token = key[27:]
+                value = key[19:27].lower()
```

Comment:
Not a big deal, but the name "value" here tripped up my brain for a second. I'm not sure what the best alternative name is, though. Maybe "subkey"?